Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -74,7 +84,7 @@ public Vector3 ManipulationHandPosition
         {
             get
             {
-                Vector3 handPosition = Vector3.zero;
```

Comment:
Moving forward a PR should only contain changes related to a specific issue.
This way it's 1) easier to review changes and also 2) easier to revert commits or merge in case something goes wrong
Just my 2c...